imports/filters/filters.go

```diff
@@ -140,6 +143,7 @@ func projectImportPath() string {
 		log.Fatal(errors.New("$GOPATH not set"))
 	}
 
-	// leverage the $GOPATH to strip out everything but the base git URL
-	return path[len(gopath+"/src/"):]
+	// trim the $GOPATH/src off the current working directory
+	gosrc := filepath.Join(gopath, "src") + string(filepath.Separator)
+	return strings.TrimPrefix(cwd, gosrc)
```

Comment:
Quick question, for the `projectImportPath` method, I think we always want to return a unix like path (since import paths are always unixy), never a windows like path.  I think this code will return a windows like path if the user is on windows, and that might mess up the `Local` filter method.